extensions/amp-carousel/0.2/test/test-type-slides.js

```diff
@@ -165,16 +165,21 @@ describes.realWin(
 
     it('should style snap for container and content correctly', async () => {
       const carousel = await getCarousel({loop: true});
-      const slideWrappers = getSlideWrappers(carousel);
-      expect(slideWrappers.length).to.equal(5);
-
-      // Ensure that the content has the snap property not wrapper
-      // or else it will break scrolling animation.
-      for (let i = 0; i < slideWrappers.length; i++) {
-        expect(slideWrappers[i].style.scrollSnapAlign).to.equal('');
-        expect(slideWrappers[i].children[0].style.scrollSnapAlign).to.not.equal(
-          ''
-        );
+      const slides = carousel.querySelector('.i-amphtml-carousel-scroll')
+        .children;
+      expect(slides.length).to.equal(20);
```

Comment:
I wonder if this was a mixed case test before. 

Not important for merge, but would this still pass? If so, I think it would be less destructive. I don't think we need to test the amount of slides either.

```suggestion
      const slideWrappers = getSlideWrappers(carousel);
      expect(slideWrappers.length).to.equal(5);
      
      const slides = carousel.querySelector('.i-amphtml-carousel-scroll')
        .children;
```